src/org/thoughtcrime/securesms/ProfileActivity.java

```diff
@@ -130,7 +131,9 @@ public boolean onCreateOptionsMenu(Menu menu) {
 
       if (chatId != 0) {
         inflater.inflate(R.menu.profile_chat, menu);
-        if (chatIsGroup) {
+        if (chatIsMailingList) {
+          menu.findItem(R.id.edit_name).setVisible(false);
```

Comment:
I am using mailing lists on my phone a little (very little, and only GitHub notifications) and I actually liked the possibility to change the group name and image (e.g. renaming `deltachat/deltachat-core-rust` to `deltachat-core-rust`).

The core will not send any mail out if the name is changed.

OTOH, this might break with the expectation that the name is changed for all users (as this is the case for all other groups).

Reply:
let's keep things simple for now. the changed name won't be synced in a multi-device-setup and it is also not helpful when user have different names for the same group.

also, this breaks the expectation, that the name change is for all users, as in normal groups.